src/lib/rest/curl_context.cpp

```diff
@@ -0,0 +1,94 @@
+#include "curl_context.h"
+
+#include <map>
+
+#include <assert.h>
+
+#include "logMsg/logMsg.h"
+#include "logMsg/traceLevels.h"
+
+
+static pthread_mutex_t contexts_mutex = PTHREAD_MUTEX_INITIALIZER;
+static std::map<std::string, struct curl_context> contexts;
+
+struct curl_context
```

Comment:
A matter of style... I think we put the return value of the function in the same line of the function name. @kzangeli could you confirm?